src/controller/python/chip/crypto/p256keypair.py

```diff
@@ -0,0 +1,71 @@
+from typing import *
+from ctypes import *
+
+from ecdsa import NIST256p, SigningKey, ECDH
+import abc
+
+from chip import native
+
+_pychip_ECDSA_sign_msg_func = CFUNCTYPE(c_bool, py_object, c_char_p, c_size_t, c_char_p, POINTER(c_uint32))
```

Comment:
suggest `c_size_t` for the size pointer returned as the last arg, here and for derive secret.

Reply:
done